test/integration/run-bats.sh

```diff
@@ -15,7 +15,7 @@ function quiet_run () {
 
 function cleanup_machines() {
     if [[ $(machine ls -q | wc -l) -ne 0 ]]; then
-        quiet_run machine rm -f $(machine ls -q)
+        quiet_run machine rm -y -f $(machine ls -q)
```

Comment:
`-y` is for automatic yes to avoid promoting confirmation to the user.  `-f` does two things, a force removal and implies a `-y` (recent change, as per the suggestion by @nathanleclaire), therefore, although when `-f` is used it does not require `-y`, it would not change the expected behaviour. Prior to the change the tests was modified to include a `-y` flag, I would happy to remove the flag if you feel its not required ?